notNeededPackages.json

```diff
@@ -2497,6 +2497,10 @@
             "libraryName": "kagekiri",
             "asOfVersion": "1.1.0"
         },
+        "kakao-js-sdk": {
+            "libraryName": "kakao-js-sdk",
+            "asOfVersion": "2.0.0"
```

Comment:
```suggestion
            "asOfVersion": "1.1.4"
```

Reply:
ops, sorry for this, didn't see the problem in versioning. How this can be addressed? Is that a case for '[skip ci]' and manual merge?